src/report/create_all_reports.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+"""Generate all reports."""
+
+# Standard Python Libraries
+import csv
+import os
+
+HOME_DIR = "/home/cisa"
+SHARED_DATA_DIR = HOME_DIR + "/shared/"
+
+
+def main():
+    """Generate all reports."""
+    with open(SHARED_DATA_DIR + "artifacts/unique-agencies.csv") as agency_csv:
+        for row in sorted(csv.reader(agency_csv)):
+            bashCommand = (
+                HOME_DIR + "/report/generate_https_scan_report.py " + '"' + row[0] + '"'
+            )
+            # generate_https_scan_report.py isn't written in a way
+            # that easily allows it to be run in any other way.  Hence
+            # the nosec.
+            os.system(bashCommand)  # nosec B605
```

Comment:
Should we look at replacing this? Per [Python documentation](https://docs.python.org/3/library/subprocess.html#replacing-os-system):
> The [`os.system()`](https://docs.python.org/3/library/os.html#os.system) function ignores SIGINT and SIGQUIT signals while the command is running, but the caller must do this separately when using the [`subprocess`](https://docs.python.org/3/library/subprocess.html#module-subprocess) module.

Reply:
Please see commit 524d807.